addons/binding/org.openhab.binding.elerotransmitterstick/README.md

```diff
@@ -0,0 +1,75 @@
+# Elero Transmitter Stick Binding
+
+Allows to control Elero rollershutters with openHAB through a connected Elero Transmitter Stick. 
+
+## Supported Things
+
+### Elero Transmitter Stick (Bridge)
+
+Represents the physical Elero Transmitter Stick connected to a USB port.
+
+### Elero Channel (Thing)
+
+Represents one of the channels of an Elero Transmitter Stick.
+
+### Elero Group (Thing)
+
+Represents a group of channels allowing to have a common status channel as well as sending a command to all channels of this group simultaneously. 
+
+## Discovery
+
+Discovery is supported only for Elero Channels. Just press the button in order to put it into your inbox after you have successfully manually created an Elero Transmitter Stick.
+
+## Thing Configuration
+
+### Elero Transmitter Stick
+
+* Port Name: The name of the port to which the stick is connected, e.g. /dev/ttyUSB0.
+* Update Interval: The number of seconds to wait before polling a single channel again.
+
+### Elero Group
+
+* Channel IDs: A comma separated list of channel ids. These have to match the channelid property of the respective Elero Channel.
+
+## Channels
+
+* Control: Rollershutter channel allowing to control the Elero Channel(s). Supports UP, DOWN, STOP and the following distinct percentages:
+
+Percentage | Rollershutter Command | Result
```

Comment:
Because these are status values and can not be used as commands.

Reply:
OK, I see. Though it is a bit confusing that it supports the status value 50 but not a command with value 50.

Reply:
The stick api does not deal with percentages. It only knows the commands UP, DOWN, STOP, INTERMEDIATE and VENTILATION and reports the current status as an enum value: Readonly channel providing a string with status information from the Elero Channel. Possible values are: NO_INFORMATION, TOP, BOTTOM, INTERMEDIATE, VENTILATION, BLOCKING, OVERHEATED, TIMEOUT, START_MOVE_UP, START_MOVE_DOWN, MOVING_UP, MOVING_DOWN, STOPPED, TOP_TILT, BOTTOM_INTERMEDIATE, SWITCHED_OFF, SWITCHED_ON

As I tried to map that to what the Rollershutter Item has/needs I mapped these statuses to percentages, which at least allows to define a rollershutter item and use the UP, DOWN, STOP commands as would be expected. Since the rollershutter does not have commands to drive the shutter to an INTERMEDIATE position, I also mapped the distinct percentages 25 and 75 as commands to the INTERMEDIATE and VENTILATION positions the stick accepts as commands.